readthedocs/projects/models.py

```diff
@@ -179,6 +181,17 @@ class AddonsConfig(TimeStampedModel):
 
     # Flyout
     flyout_enabled = models.BooleanField(default=True)
+    flyout_sorting = models.CharField(
+        choices=ADDONS_FLYOUT_SORTING_CHOICES,
+        default=ADDONS_FLYOUT_SORTING_LEXICOGRAPHYCALLY,
+    )
+    flyout_sorting_custom_pattern = models.CharField(
+        max_length=32,
+        default=None,
+        null=True,
+        blank=True,
+        help_text="Sorting pattern supported by BumpVer",
```

Comment:
This should probably be a link, if we're going to call out `BumpVer` explicitly.

Reply:
Yes. I think we will eventually document this by ourselves, but I think it's fine for now to provide a link to users so they know where to find out this information.